index.js

```diff
@@ -81,20 +85,17 @@ module.exports = {
     var rollupReplaceConfig = {
       include: ['**/ember-service-worker/**/*.js'],
       delimiters: ['{{', '}}'],
-      ROOT_URL: rootURL
-    };
-
-    // define `BUILD_TIME` as a getter so that each time
-    // rollup runs the version string is changed.  Otherwise,
-    // when running `ember s` this would never change (leading to
-    // cache invalidation trollage).
-    Object.defineProperty(rollupReplaceConfig, 'BUILD_TIME', {
-      enumerable: true,
-      configurable: true,
-      get: function() {
+      ROOT_URL: rootURL,
+      PROJECT_REVISION: this._projectVersion,
+
+      // define `BUILD_TIME` as a getter so that each time
+      // rollup runs the version string is changed.  Otherwise,
+      // when running `ember s` this would never change (leading to
+      // cache invalidation trollage).
+      get BUILD_TIME() {
```

Comment:
FYI - I did verify that this works in node 0.10:

![tmux attach tmux 2016-07-18 10-06-53](https://cloud.githubusercontent.com/assets/12637/16917333/317c6ca0-4ccf-11e6-9cd5-6fda64781560.png)